pkgs/games/frogatto/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, buildEnv, stdenv, callPackage, makeWrapper, makeDesktopItem }:
+
+let
+  description = "Action-adventure game, starring a certain quixotic frog";
+  engine = callPackage ./engine.nix { };
+  data = callPackage ./data.nix { };
+  desktopItem = makeDesktopItem {
+    name = "frogatto";
+    exec = "frogatto";
+    startupNotify = "true";
+    icon = "${data}/share/frogatto/modules/frogatto/images/os/frogatto-icon.png";
+    comment = description;
+    desktopName = "Frogatto";
+    genericName = "frogatto";
+    categories = "Application;Game;ArcadeGame;";
+  };
+  version = "unstable-2018-12-18";
+in buildEnv rec {
+  name = "frogatto-${version}";
+
+  buildInputs = [ makeWrapper ];
+  paths = [ engine data desktopItem ];
+  pathsToLink = [ "/bin" "/share/frogatto/data" "/share/frogatto/images" "/share/frogatto/modules" ];
```

I just remembered that since we're sticking with using `buildEnv` that we can just add
```suggestion
  pathsToLink = [ "/bin" "/share/applications" "/share/frogatto/data" "/share/frogatto/images" "/share/frogatto/modules" ];
```
to link the `desktopItem` when it's in `paths` and we can remove the thing in `postBuild`.